bundles/AdminBundle/Controller/Admin/Asset/AssetHelperController.php

```diff
@@ -326,10 +319,6 @@ protected function getFieldGridConfig($field, $language = '', $keyPrefix = null)
         } elseif (in_array($fieldDef[0], $defaulMetadataFields)) {
             $type = 'input';
         } else {
-            //check if predefined metadata exists, otherwise ignore
```

Comment:
this should be reverted to avoid situations when you change or delete predefined metadata then grid operations doesn't work as expected like update or displaying values so it is better to skip the column in that case

Reply:
done